src/app/static/src/tests/metadata/actions.test.ts

```diff
@@ -0,0 +1,41 @@
+import {
+    mockAxios,
+    mockFailure,
+    mockSuccess
+} from "../mocks";
+import {actions} from "../../app/store/metadata/actions";
+
+describe("Metadata actions", () => {
+
+    beforeEach(() => {
+        // stop apiService logging to console
+        console.log = jest.fn();
+        mockAxios.reset();
+    });
+
+    afterEach(() => {
+        (console.log as jest.Mock).mockClear();
+    });
+
+    it("commits metadata after successful fetch", async () => {
+
+        mockAxios.onGet(`/meta/plotting/Malawi`)
+            .reply(200, mockSuccess("TEST DATA"));
+
+        const commit = jest.fn();
+        await actions.getPlottingMetadata({commit} as any, "Malawi");
+
+        expect(commit.mock.calls[0][0]).toStrictEqual({type: "PlottingMetadataFetched", payload: "TEST DATA"});
+    });
+
+    it("commits error after unsuccessful fetch", async () => {
+
+        mockAxios.onGet(`/meta/plotting/Malawi`)
+            .reply(500, mockFailure("Test Error"));
```

Comment:
If response errors in API due to something like missing country in the config it will return a 400 error. Perhaps we should change that to return 500 instead? Either way - do we need to test for a 400 error here? or is the handling the same for 400 type and 500 type?

Reply:
Should be handled in the same way, but I'll add a test in for 400 too.